### PR TITLE
Environment monitoring install guide

### PR DESCRIPTION
**Pull request by simonmicroshare**
New Environment Monitoring Installation Guide page

### CODE REVIEW

File: docs/2/installer/specific-installations/environment-monitoring-install.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+| **1.** **Activating the sensor** <br>- Our wireless sensors are pre-installed with batteries. To activate pull the shipping isolating battery tab from the sensor.<br>-	The LED on the sensor should begin to flash intermittently.<br> | {% include image.html url="/assets/img/install_environment/EnvironmentSensor.jpg" description="Environment Sensor" %} |
+| **2.** **Preparing the sensor for fixing** <br><br>- Using a suitable adhesive tape, **ensure that you place the tape on the opposite side of the QR code, covering the QR code will prevent you from using the Deploy-M app to add your device to the system.** | {% include image.html url="/assets/img/install_environment/tabs_side_adhesive.png" description="Sensor with adhesive tape" %}|
+| **3.** **Registering the sensor using the Deploy-M app [iOS & Android]** <br><br>- You should already have the Deploy-M app and login details as part of your project set-up with Microshare<br>- Login using your credentials, if you are unsure or require assistance, please contact [installation@microshare.io](mailto:installation@microshare.io) | {% include image.html url="/assets/img/leak-installation/Image12.png" description="Banner" %} |
```
Comment (JulienBardin) [inline docs/2/installer/specific-installations/environment-monitoring-install.md:46]: It seems like you kept the leak image here

Is it ok ? I know it's for Deploy-M

File: _data/docs.yml [hunk 2]
```diff
@@ -90,6 +90,8 @@
       keywords : ["fridge monitoring","fridge"]
     - title: Leak Detector Installation Guide
       keywords : ["leak","water"]
+    - title: Environment Monitoring Installation
+      keywords : ["environment","environment monitoring","install"
```
Comment (JulienBardin) [inline _data/docs.yml:94]: It might be interesting in this "specific installation" group to keep the installations in alphabetic order

Comment (simonmicroshare) [reply _data/docs.yml:94]: Good idea Julien

File: docs/2/installer/specific-installations/environment-monitoring-install.md (hunk 1 above)
Comment (simonmicroshare) [reply docs/2/installer/specific-installations/environment-monitoring-install.md:46]: Yes, this is correct Julien, we may update in the future, but for now we are using the same images so I thought this was more efficient....

File: _data/docs.yml (hunk 2 above)
Comment (JulienBardin) [reply _data/docs.yml:94]: @tidelbo

**Pull request merged**